### ISSUE TITLE
Null geometry repr bug

### ISSUE THREAD

**Issue by wurenzhe163** [aazuspan/eerepr#16]
`Map.draw_features[0]`

```
AttributeError                            Traceback (most recent call last)
File D:\Code_base\anaconda\envs\GEE\lib\site-packages\IPython\core\formatters.py:342, in BaseFormatter.__call__(self, obj)
    340     method = get_real_method(obj, self.print_method)
    341     if method is not None:
--> 342         return method()
    343     return None
    344 else:

File D:\Code_base\anaconda\envs\GEE\lib\site-packages\eerepr\repr.py:82, in _ee_repr(obj)
     77 if _is_nondeterministic(obj):
     78     # We don't want to cache nondeterministic objects, so we'll add add a unique attribute
     79     # that causes ee.ComputedObject.__eq__ to return False, preventing a cache hit.
     80     setattr(obj, "_eerepr_id", uuid.uuid4())
---> 82 rep = _repr_html_(obj)
     83 mbs = len(rep) / 1e6
     84 if mbs > options.max_repr_mbs:

File D:\Code_base\anaconda\envs\GEE\lib\site-packages\eerepr\repr.py:62, in _repr_html_(obj)
     60 css = _load_css()
     61 js = _load_js()
---> 62 body = convert_to_html(info)
     64 return (
     65     "<div>"
     66     f"<style>{css}</style>"
   (...)
     71     "</div>"
     72 )

File D:\Code_base\anaconda\envs\GEE\lib\site-packages\eerepr\html.py:32, in convert_to_html(obj, key)
     30     return list_to_html(obj, key)
     31 elif isinstance(obj, dict):
---> 32     return dict_to_html(obj, key)
     34 key_html = f"<span class='ee-k'>{key}:</span>" if key is not None else ""
     35 return (
     36     "<li>"
     37     f"{key_html}"
     38     f"<span class='ee-v'>{obj}</span>"
     39     "</li>"
     40 )

File D:\Code_base\anaconda\envs\GEE\lib\site-packages\eerepr\html.py:58, in dict_to_html(obj, key)
     56 """Convert a Python dictionary to an HTML <li> element."""
     57 obj = _sort_dict(obj)
---> 58 label = _build_label(obj)
     60 header = f"{key}: " if key is not None else ""
     61 header += label

File D:\Code_base\anaconda\envs\GEE\lib\site-packages\eerepr\html.py:249, in _build_label(obj)
    246 if obj_type not in labelers:
    247     obj_type = "_Typed"
--> 249 return labelers[obj_type](obj)

File D:\Code_base\anaconda\envs\GEE\lib\site-packages\eerepr\html.py:113, in _build_feature_label(obj)
    111 def _build_feature_label(obj: dict) -> str:
    112     n = len(obj.get("properties", []))
--> 113     geom_type = obj.get("geometry", {}).get("type")
    114     type_label = f"{geom_type}, " if geom_type is not None else ""
    115     noun = "property" if n == 1 else "properties"

AttributeError: 'NoneType' object has no attribute 'get'

<ee.feature.Feature at 0x23a08257cd0>
```

**Comment by aazuspan** [comment]
Thanks for the report. This is a bug in `eerepr` if you try to print a feature with no geometry. I'll work on fixing it.

Note to self, you can reproduce with:

```python
ee.Feature(None)._repr_html_()
```

**Comment by wurenzhe163** [comment]
Thanks very much, eerepr has given me a great convenience!

**Comment by aazuspan** [comment]
Glad to hear it! I released the fix with `0.0.4`. You can update with `pip install --upgrade eerepr`.
